docs/tailwind-best-practices.md

```diff
@@ -0,0 +1,177 @@
+# Tailwind CSS Best Practices
+
+A comprehensive guide for using Tailwind effectively and consistently across our design system monorepo and engineering organization.
```

Comment:
We may move this documentation to https://github.com/MetaMask/contributor-docs if it makes sense to in future, for now this is a mix of our conventions that should carry over to other consuming projects